clumio/provider_test.go

```diff
@@ -46,6 +48,15 @@ func testAccPreCheck(t *testing.T) {
 	}
 }
 
+// testAccPreCheckClumio validates that the required environment variables are set before
+// the acceptance test is executed.
+func testAccPreCheckClumio(t *testing.T) {
+	testFailIfEmpty(t, clumioTestAwsAccountId, clumioTestAwsAccountId+" cannot be empty")
+	testFailIfEmpty(t, clumioApiToken, clumioApiToken+" cannot be empty.")
+	testFailIfEmpty(t, clumioApiBaseUrl, clumioApiBaseUrl+" cannot be empty.")
```

Comment:
Not sure I understand what this test is saying ... does it mean that the API Token and API Base URL "must" be given and cannot be empty? If so, I thought for backwards-compatibility, they could be left empty?

Reply:
This is only for running the acceptance test. For running the acceptance tests of this resource, these are mandatory values.